tests/integration_tests/data_steward/cdr_cleaner/cleaning_rules/identifying_field_suppression_test.py

```diff
@@ -167,10 +167,8 @@ def test_identifying_field_suppression(self):
                     'provider_id', 'care_site_id'
                 ],
                 'loaded_ids': [12345, 6789],
-                'cleaned_values': [
-                    (12345, 1, 1990, None, None, None, None, None),
-                    (6789, 2, 1980, None, None, None, None, None)
-                ]
+                'cleaned_values': [(12345, 1, 1990, 0, 0, 0, 0, 0),
+                                   (6789, 2, 1980, 0, 0, 0, 0, 0)]
```

Comment:
similar to above, these are all nullable fields.  during deidentification, we set nullable fields to `null` if the field data is to be removed.  This test does not seem to meet that expectation.

Reply:
As with above, when running the integration tests they were failing due to `None` not being a valid value.  I don't remember if it was due to the column definition itself in the test tables or due to the way the tests themselves are run.